index.html

```diff
@@ -9,88 +9,103 @@
 		<script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script>
 	<![endif]-->
 </head>
-
 <body>
+	<div class="container">
+<!-- Menu Content  -->
+		<header>
+		S&J
+			<nav>
+			Home
+			Services
+			Contact
+			</nav>
+		</header>
 
-	<h1>You got this!  Good luck.</h1>
-
-<!-- Menu Content 
-S&J
-
-Home
-Services
-Contact
--->
-
-
-<!-- Home Page Content:
-	
-Integrity,
-Excellence,
-Progress.
+<!-- Home Page Content -->
+		<section class="cta">
+			<div class="cta-text">
+			<h1>Integrity,
```

Comment:
great work on the h1. always make sure that every section has a h1 or similar heading tag even if it is not to be saw by the end user. remember that you can always move the text off screen by using things like indent css property